tests/Hangfire.PostgreSql.Tests/Utils/ConnectionUtils.cs

```diff
@@ -16,7 +15,7 @@ public static class ConnectionUtils
     private const string DefaultDatabaseName = @"hangfire_tests";
     private const string DefaultSchemaName = @"hangfire";
 
-    private const string DefaultConnectionStringTemplate = @"Server=127.0.0.1;Port=5432;Database=postgres;User Id=postgres;Password=password;";
+    private const string DefaultConnectionStringTemplate = @"Server=127.0.0.1;Port=5432;Database=postgres;User Id=admin;Password=admin;";
```

Comment:
Revert.

Reply:
Done.

Reply:
Close, not quite. Password should be password, not postgres.

Reply:
Oh, you are right. Sorry.
Now should be fully reverted.